tritium-core/src/main/java/com/palantir/tritium/event/CompositeInvocationEventHandler.java

```diff
@@ -134,11 +135,11 @@ private static void preInvocationFailed(
     }
 
     private static void handleSuccess(
-            @Nullable InvocationEventHandler<?> handler,
+            InvocationEventHandler<?> handler,
             @Nullable InvocationContext context,
             @Nullable Object result) {
         try {
-            if (handler != null) {
+            if (context != null) {
```

Comment:
wondering if we should pull the follow up out of the `try`?

```suggestion
        if (context == null) {
            return;
        }
```

Reply:
I was thinking the same thing, but didn't want to put too much into one PR. Might as well though.